network-api/networkapi/wagtailpages/translation.py

```diff
@@ -249,3 +254,32 @@ class YoutubeRegretsReporterPageTR(TranslationOptions):
         'intro_text',
         'intro_images',
     }
+
+
+@register(ProductPage)
+class ProductPageTR(TranslationOptions):
+    fields = (
+        'company',
+        'blurb',
+        'price',
+        'worst_case',
+        'signup_requirement_explanation',
+        'how_does_it_use_data_collected',
+        'security_updates_helptext',
+        'strong_password_helptext',
+        'manage_vulnerabilities_helptext',
+        'privacy_policy_helptext',
+        'phone_number',
+        'live_chat',
+        'email',
```

Comment:
let's keep this list as short as possible until the work is done, so we can add "all the fields" as part of the last "okay these are the things we'll translate" migration.

```suggestion
        'name',
        'company',
        'price',
```